### DISCUSSION TITLE
Feedback request: Should variables and secrets have an optional "description" field?

### DISCUSSION THREAD

**Discussion by juliandunn** [community/community#29856] [Actions]
👋 Hello again, Julian from the Actions product team at GitHub. As you may know from [this thread](https://github.com/orgs/community/discussions/24988#discussioncomment-3304864), we are working on a feature to allow non-secret configuration data ("variables"), so that secrets are not the only way to parameterize workflows.

We are considering adding an optional _description_ field for both secrets and variables. I am hoping to get the community's feedback on whether this would be valuable or not, and why. _(For further context, a member of our team has asked why we need to add a description field -- "can't customers just name secrets and variables in a descriptive manner?")_

Please drop your comments below!

**Comment by radar-blip** [comment]
A short optional description cant hurt.

**Comment by airtower-luna** [comment]
In programming I'm all for descriptive names, but that's no replacement for comments. Same principle applies here. :wink:
